HTTPServerProject/HTTPServerProxy/Response.cs

```diff
@@ -0,0 +1,124 @@
+using HTTPServerRead.Streams;
+using HTTPServerRead.Header;
+using HTTPServerRead.Body;
+using HTTPServerWrite.Response;
+using HTTPServerWrite.Streams;
+
+namespace HTTPServerProxy.Response;
+
+public class ProxyResponse
+{
+    Header _header;
+    Body _body;
+    IReadStreams _reader;
+    IWriteStreams _writer;
+    string _path;
+    string _type;
+
+    public ProxyResponse(IReadStreams reader, IWriteStreams writer, string path, string type)
+    {
+        _reader = reader;
+        _writer = writer;
+        _header = new Header(_reader);
+        _body = new Body(_reader);
+        _path = path;
+        _type = type;
+    }
+
+    public void GetResponse()
+    {
+        var code = GetStatusCode();
+        var headers = GetHeaders();
+        var body = GetBody();
+        
+        if (_type == "POST")
+        {
+            if (code == 200)
+            {
+                GetFullJSONContentType(headers);
+                if (body.StartsWith('{') && body.EndsWith('}') && body.Length > 2)
+                {
+                    Console.WriteLine(body);
```

Comment:
Were these just used for debugging? It's probably good to remove these if so

Reply:
Agreed.